File: Groupe2/TP1/src/boucles.py
# Demande la taille du triangle
# taille = int(input("Entrez la taille du triangle (inférieure à 10) : "))

# # Validation de la taille
# if taille >= 10:
#     print("La taille doit être inférieure à 10.")
# else:
#     # Affichage du triangle avec des boucles for
#     for i in range(1, taille + 1):
#         for j in range(1, i + 1):
#             if j % 2 == 1:
#                 print("*", end=" ")
#             else:
#                 print("#", end=" ")
#         print()

# Demande la taille du triangle
taille = int(input("Entrez la taille du triangle (inférieure à 10) : "))

# Validation de la taille
if taille >= 10:
    print("La taille doit être inférieure à 10.")
else:
    # Affichage du triangle avec une boucle while
    i = 1
    while i <= taille:
        j = 1
        while j <= i:
            if j % 2 == 1:
                print("*", end=" ")
            else:
                print("#", end=" ")
            j += 1
        print()
        i += 1

File: Groupe2/TP1/src/boucles.c
#include <stdio.h>

// int main() {
//     int taille;
    
//     // Demande la taille du triangle
//     printf("Entrez la taille du triangle (inferieure a 10) : ");
//     scanf("%d", &taille);

//     // Validation de la taille
//     if (taille >= 10) {
//         printf("La taille doit etre inferieure a 10.\n");
//         return 1;
//     }

//     // Affichage du triangle avec une boucle for
//     for (int i = 1; i <= taille; i++) {
//         for (int j = 1; j <= i; j++) {
//             if (j % 2 == 1) {
//                 printf("* ");
//             } else {
//                 printf("# ");
//             }
//         }
//         printf("\n");
//     }

//     return 0;
// }

int main() {
    int taille;
    int i, j;
    
    // Demande la taille du triangle
    printf("Entrez la taille du triangle (inferieure a 10) : ");
    scanf("%d", &taille);

    // Validation de la taille
    if (taille >= 10) {
        printf("La taille doit etre inferieure a 10.\n");
        return 1;
    }

    // Affichage du triangle avec une boucle while
    i = 1;
    while (i <= taille) {
        j = 1;
        while (j <= i) {
            if (j % 2 == 1) {
                printf("* ");
            } else {
                printf("# ");
            }
            j++;
        }
        printf("\n");
        i++;
    }

    return 0;
}

